In [6]:
import os
from PublicDataReader import TransactionPrice

service_key = os.getenv("PUBLIC_DATA_HUB")


api = TransactionPrice(service_key)

## 국토교통부 실거래가 정보 조회 서비스

<div align="center">

| **서비스명**                          | **부동산 유형** | **거래 유형** |
| ------------------------------------- | ------------ | ------------ |
| [국토교통부_아파트 매매 실거래가 상세 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126468) | 아파트       | 매매         |
| [국토교통부_아파트 전월세 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126474) | 아파트       | 전월세       |
| [국토교통부_아파트 분양권전매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126471) | 분양입주권   | 매매         |
| [국토교통부_오피스텔 매매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126464)  | 오피스텔     | 매매         |
| [국토교통부_오피스텔 전월세 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126475) | 오피스텔     | 전월세       |
| [국토교통부_연립다세대 매매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126467) | 연립다세대   | 매매         |
| [국토교통부_연립다세대 전월세 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126473) | 연립다세대   | 전월세       |
| [국토교통부_단독/다가구 매매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126465) | 단독다가구   | 매매         |
| [국토교통부_단독/다가구 전월세 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126472) | 단독다가구   | 전월세       |
| [국토교통부_토지 매매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126466) | 토지         | 매매         |
| [국토교통부_상업업무용 부동산 매매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126463) | 상업업무용   | 매매         |
| [국토교통부_공장 및 창고 등 부동산 매매 실거래가 자료](https://www.data.go.kr/tcs/dss/selectApiDataDetailView.do?publicDataPk=15126470) | 공장창고등   | 매매         |

</div>


## 입력 명세

* [아파트매매 실거래 상세 자료 조회 서비스](#아파트매매-실거래-상세-자료-조회-서비스)
* [아파트 전월세 자료 조회 서비스](#아파트-전월세-자료-조회-서비스)
* [아파트 분양권전매 신고 자료 조회 서비스](#아파트-분양권전매-신고-자료-조회-서비스)
* [오피스텔 매매 신고 조회 서비스](#오피스텔-매매-신고-조회-서비스)
* [오피스텔 전월세 신고 조회 서비스](#오피스텔-전월세-신고-조회-서비스)
* [연립다세대 매매 실거래자료 조회 서비스](#연립다세대-매매-실거래자료-조회-서비스)
* [연립다세대 전월세 실거래자료 조회 서비스](#연립다세대-전월세-실거래자료-조회-서비스)
* [단독/다가구 매매 실거래 조회 서비스](#단독다가구-매매-실거래-조회-서비스)
* [단독/다가구 전월세 자료 조회 서비스](#단독다가구-전월세-자료-조회-서비스)
* [토지 매매 신고 조회 서비스](#토지-매매-신고-조회-서비스)
* [상업업무용 부동산 매매 신고 자료 조회 서비스](#상업업무용-부동산-매매-신고-자료-조회-서비스)
* [공장 및 창고 등 부동산 매매 신고 자료 조회 서비스](#공장-및-창고-등-부동산-매매-신고-자료-조회-서비스)

<div align="center">

| 이름             | 설명                                                                                                                               | 데이터 타입   | 샘플 데이터   | 항목구분    |
|:-----------------|:-----------------------------------------------------------------------------------------------------------------------------------|:--------------|:--------------|:------------|
| property_type    | 부동산 유형<br>(아파트, 오피스텔, 단독다가구, 연립다세대, 토지, 분양입주권, 공장창고등)                                            | String        | 아파트        | 필수        |
| trade_type       | 거래 유형<br>(매매, 전월세)                                                                                                        | String        | 매매          | 필수        |
| sigungu_code     | 시군구의 5자리 지역코드<br>(서울 서초구: 11650, 경기 성남 분당구: 41135)                                                           | String        | 11650         | 필수        |
| year_month       | 조회 년월 (단일 월 조회 시 필수)<br>(2023년 1월: 202301)<br>※ start_year_month와 end_year_month 모두 입력 시 기간 내 조회가 실행됨 | String        | 202301        | 조건부 필수 |
| start_year_month | 조회 시작 년월 (기간 내 조회 시 필수)<br>(2022년 1월 202201)                                                                       | String        | 202201        | 조건부 필수 |
| end_year_month   | 조회 종료 년월 (기간 내 조회 시 필수)<br>(2022년 12월: 202212)                                                                     | String        | 202212        | 조건부 필수 |
| verbose          | 데이터 조회 진행 상황 메시지 출력 여부<br>(출력: True, 미출력: False)<br>※ 기본값: False                                           | Boolean       | True          | 선택        |

</div>


## 아파트매매 실거래 상세 자료 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명        | 설명                    | 샘플데이터      |
|:-----------|:----------------------|:-----------|
| 법정동시군구코드   | 법정동시군구코드              | 11110      |
| 법정동읍면동코드   | 법정동읍면동코드              | 송인동        |
| 법정동지번코드    | 법정동지번코드               | 1          |
| 법정동본번코드    | 법정동본번코드               | 0202       |
| 법정동부번코드    | 법정동부번코드               | 0003       |
| 도로명        | 도로명                   | 종로66길      |
| 도로명시군구코드   | 도로명시군구코드              | 11110      |
| 도로명코드      | 도로명코드                 | 4100372    |
| 도로명일련번호코드  | 도로명일련번호코드             | 01         |
| 도로명지상지하코드  | 도로명지상지하코드             | 0          |
| 도로명건물본번호코드 | 도로명건물본번호코드            | 00028      |
| 도로명건물부번호코드 | 도로명건물부번호코드            | 00000      |
| 법정동        | 법정동                   | 17500      |
| 단지명        | 단지명                   | 종로중흥S클래스   |
| 지번         | 지번                    | 202-3      |
| 전용면적       | 전용면적                  | 17.811     |
| 계약년도       | 계약년도                  | 2024       |
| 계약월        | 계약월                   | 7          |
| 계약일        | 계약일                   | 23         |
| 거래금액       | 거래금액(만원)              | 12,000     |
| 층          | 층                     | 10         |
| 건축년도       | 건축년도                  | 2013       |
| 단지일련번호     | 단지 일련번호               | 11110-2339 |
| 해제여부       | 해제여부                  |            |
| 해제사유발생일    | 해제사유발생일               |            |
| 거래유형       | 거래유형(중개및직거래여부)        | 중개거래       |
| 중개사소재지     | 중개사소재지(시군구단위)         | 서울 종로구     |
| 등기일자       | 등기일자                  |            |
| 아파트동명      | 아파트 동명                |            |
| 매도자        | 거래주체정보(개인/법인/공공기관/기타) | 개인         |
| 매수자        | 거래주체정보(개인/법인/공공기관/기타) | 개인         |
| 토지임대부아파트여부 | 토지임대부 아파트 여부          | N          |


In [ ]:
df = api.get_data(
    property_type="아파트",
    trade_type="매매",
    sigungu_code="11650",
    year_month="202212",
)

# 기간 내 조회
df = api.get_data(
    property_type="아파트",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

---

## 아파트 전월세 자료 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명           | 샘플데이터   |
|:---------------|:---------------|:-------------|
| 지역코드       | 지역코드       | 11110        |
| 법정동         | 법정동         | 창신동       |
| 지번           | 지번           | 703          |
| 아파트         | 아파트명       | 창신쌍용2    |
| 건축년도       | 건축년도       | 1993         |
| 층             | 층             | 3            |
| 전용면적       | 전용면적(㎡)   | 54.7         |
| 년             | 계약년도       | 2022         |
| 월             | 계약월         | 1            |
| 일             | 일             | 26           |
| 보증금액       | 보증금액(만원) | 28300        |
| 월세금액       | 월세금액(만원) | 0            |
| 계약구분       | 계약구분       | 갱신         |
| 계약기간       | 계약기간       | 22.02~24.02  |
| 갱신요구권사용 | 갱신요구권사용 | 사용         |
| 종전계약보증금 | 종전계약보증금 | 27000        |
| 종전계약월세   | 종전계약월세   | 0            |

In [7]:
# 단일 월 조회
df = api.get_data(
    property_type="아파트",
    trade_type="전월세",
    sigungu_code="11650",
    year_month="202212",
)

# 기간 내 조회
df = api.get_data(
    property_type="아파트",
    trade_type="전월세",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  법정동          단지명       지번    전용면적  계약년도  계약월  계약일    보증금액  \
15144    11650  서초동        우성아파트  1326-17   59.93  2022   12   17   66150   
15145    11650  서초동   신동아아파트(1차)     1333  101.60  2022   12    3    3000   
15146    11650  서초동  서초동대우디오빌프라임  1337-22   64.05  2022   12   16   60000   
15147    11650  서초동    래미안서초에스티지  1332-13   83.60  2022   12    3  121275   
15148    11650  서초동     래미안 리더스원     1755  114.94  2022   12    5  200000   

       월세금액   층  건축년도         계약기간 계약구분 갱신요구권사용  종전계약보증금  종전계약월세  
15144     0   8  1998  23.02~25.02   갱신      사용    63000    <NA>  
15145   150  10  1978  23.01~25.01   신규    None     <NA>    <NA>  
15146     0  15  2005  23.02~25.02   갱신    None    57000    <NA>  
15147     0  22  2016  22.12~24.12   갱신    None   120000    <NA>  
15148     0  21  2020  23.02~25.02   신규    None     <NA>    <NA>

## 아파트 분양권전매 신고 자료 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명                | 샘플데이터      |
|:---------------|:--------------------|:----------------|
| 지역코드       | 지역코드            | 11650           |
| 시군구         | 물건 소재 시군구    | 서초구          |
| 법정동         | 물건 소재 법정동    | 반포동          |
| 지번           | 물건 소재 지번      | 1-1             |
| 단지           | 분양권 단지명       | 래미안 원베일리 |
| 층             | 거래물건 층         | 17              |
| 전용면적       | 전용면적            | 59.96           |
| 구분           | 분양권/입주권 구분  | 입              |
| 년             | 계약년도            | 2021            |
| 월             | 계약월              | 8               |
| 일             | 계약일              | 7               |
| 거래금액       | 거래금액            | 253948          |
| 거래유형       | 중개 및 직거래 여부 | nan             |
| 중개사소재지   | 시군구 단위         | nan             |
| 해제사유발생일 | 해제사유발생일      | 21.10.07        |
| 해제여부       | 해제여부            | O               |

In [8]:
# 단일 월 조회
# df = api.get_data(
#     property_type="분양입주권",
#     trade_type="매매",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="분양입주권",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  시군구  법정동     지번            단지명   층     전용면적    구분  계약년도  계약월  계약일  \
3    11650  서초구  잠원동   27-2          멀버리힐스  13  30.7260  None  2022    9   23   
4    11650  서초구  잠원동   27-2          멀버리힐스  12  36.5219  None  2022   10    6   
5    11650  서초구  반포동    1-1       래미안 원베일리   9  84.9500     입  2022   11   23   
6    11650  서초구  양재동  11-11  강남월드메르디앙프레스티지   6  43.2800  None  2022   12   19   
7    11650  서초구  반포동    1-1       래미안 원베일리  13  84.9800     입  2022   12    9   

     거래금액  해제여부 해제사유발생일  거래유형  중개사소재지   매도자   매수자  
3   86500  None    None  중개거래  서울 서초구  None  None  
4   85500  None    None  중개거래  서울 서초구  None  None  
5  300340  None    None  중개거래  서울 서초구  None  None  
6  118790  None    None   직거래    None  None  None  
7  320000  None    None  중개거래  서울 서초구  None  None

---

## 오피스텔 매매 신고 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 항목설명            | 샘플데이터       |
|:---------------|:--------------------|:-----------------|
| 지역코드       | 지역코드            | 11650            |
| 시군구         | 시군구              | 서초구           |
| 법정동         | 법정동              | 서초동           |
| 지번           | 지번                | 1308-26          |
| 단지           | 단지명              | 강남역리가스퀘어 |
| 건축년도       | 건축년도            | 2012             |
| 층             | 층                  | 5                |
| 전용면적       | 전용면적(㎡)        | 36.93            |
| 년             | 계약년도            | 2022             |
| 월             | 계약월              | 11               |
| 일             | 일                  | 26               |
| 거래금액       | 거래금액(만원)      | 31100            |
| 거래유형       | 중개 및 직거래 여부 | 중개거래         |
| 중개사소재지   | 시군구 단위         | 서울 서초구      |
| 해제사유발생일 | 해제사유발생일      | 22.12.02         |
| 해제여부       | 해제여부            | O                |

In [9]:
# # 단일 월 조회
# df = api.get_data(
#     property_type="오피스텔",
#     trade_type="매매",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="오피스텔",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  시군구  법정동       지번        단지명    전용면적  계약년도  계약월  계약일   거래금액   층  \
808    11650  서초구  양재동    275-1     삼호물산빌딩  27.360  2022   12    1  12800  15   
809    11650  서초구  서초동   1623-3    피앤디 솔로빌  15.760  2022   12    8  12600   6   
810    11650  서초구  우면동      759  서초 에비뉴클래식  18.470  2022   12    5  14000   8   
811    11650  서초구  서초동  1337-21   현대썬앤빌 서초  19.752  2022   12    5  20000   2   
812    11650  서초구  서초동  1337-21   현대썬앤빌 서초  18.090  2022   12    5  20000   2   

     건축년도  해제여부 해제사유발생일  거래유형          중개사소재지   매도자   매수자  
808  1991  None    None  중개거래          서울 서초구  None  None  
809  2005  None    None  중개거래          서울 서초구  None  None  
810  2015  None    None  중개거래          서울 서초구  None  None  
811  2015  None    None  중개거래  서울 강남구, 서울 서초구  None  None  
812  2015  None    None  중개거래  서울 강남구, 서울 서초구  None  None

---

## 오피스텔 전월세 신고 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명           | 샘플데이터    |
|:---------------|:---------------|:--------------|
| 지역코드       | 지역코드       | 11680         |
| 시군구         | 시군구         | 강남구        |
| 법정동         | 법정동         | 역삼동        |
| 지번           | 지번           | 832-3         |
| 단지           | 단지           | 강남역 쉐르빌 |
| 건축년도       | 건축년도       | 2014          |
| 층             | 층             | 18            |
| 전용면적       | 전용면적(㎡)   | 26.08         |
| 년             | 계약년도       | 2022          |
| 월             | 계약월         | 10            |
| 일             | 일             | 6             |
| 보증금         | 보증금(만원)   | 25500         |
| 월세           | 월세(만원)     | 0             |
| 계약구분       | 계약구분       | 갱신          |
| 계약기간       | 계약기간       | 22.12~23.12   |
| 갱신요구권사용 | 갱신요구권사용 | 사용          |
| 종전계약보증금 | 종전계약보증금 | 25500         |
| 종전계약월세   | 종전계약월세   | 0             |

In [10]:


# 단일 월 조회
# df = api.get_data(
#     property_type="오피스텔",
#     trade_type="전월세",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="오피스텔",
    trade_type="전월세",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  시군구  법정동       지번        단지명   전용면적  계약년도  계약월  계약일   보증금액  \
2250    11650  서초구  우면동      759  서초 에비뉴클래식  19.75  2022   12    1  14175   
2251    11650  서초구  서초동   1337-2   아크로텔 강남역  19.40  2022   12    1  20000   
2252    11650  서초구  방배동   1031-3   (1031-3)  28.80  2022   12    1  17000   
2253    11650  서초구  서초동  1337-21   현대썬앤빌 서초  19.42  2022   12    1   1000   
2254    11650  서초구  서초동   1337-2   아크로텔 강남역  20.72  2022   12   19   1000   

      월세금액   층  건축년도  계약구분         계약기간 갱신요구권사용  종전계약보증금  종전계약월세  
2250     0   4  2015    신규  22.12~24.12    None     <NA>    <NA>  
2251     0  18  2016    신규  23.01~24.01    None     <NA>    <NA>  
2252    10   2  1987  None         None    None     <NA>    <NA>  
2253    88   6  2015    신규  22.12~23.12    None     <NA>    <NA>  
2254    80  16  2016    갱신  22.12~23.12    None     1000      80

---

## 연립다세대 매매 실거래자료 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명                | 샘플데이터   |
|:---------------|:--------------------|:-------------|
| 지역코드       | 지역코드            | 11110        |
| 법정동         | 법정동              | 사직동       |
| 지번           | 지번                | 9            |
| 연립다세대     | 연립다세대명        | 청운빌라B동  |
| 건축년도       | 건축년도            | 2015         |
| 층             | 층                  | 11           |
| 대지권면적     | 대지권면적          | 57.43        |
| 전용면적       | 전용면적(㎡)        | 94.51        |
| 년             | 계약년도            | 2015         |
| 월             | 계약월              | 12           |
| 일             | 일                  | 1            |
| 거래금액       | 거래금액(만원)      | 82500        |
| 거래유형       | 중개 및 직거래 여부 | 중개거래     |
| 중개사소재지   | 시군구 단위         | 서울 서초구  |
| 해제사유발생일 | 해제사유발생일      | 21.01.27     |
| 해제여부       | 해제여부            | O            |

In [11]:


# 단일 월 조회
# df = api.get_data(
#     property_type="연립다세대",
#     trade_type="매매",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="연립다세대",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  법정동       단지명       지번  건축년도   전용면적    대지권면적  계약년도  계약월  계약일  \
775    11650  양재동  (344-12)   344-12  2014  20.71  13.4840  2022   12   20   
776    11650  서초동       서화빌  1564-15  2016  26.23  18.6700  2022   12   22   
777    11650  방배동      HK빌라    868-8  2001  76.67  35.4100  2022   12   15   
778    11650  서초동     진흥펠리체   1549-2  2016  29.96  19.7600  2022   12    2   
779    11650  방배동  반포스테이리버뷰    750-9  2019  17.02   5.3205  2022   12    3   

      거래금액  층  해제여부   해제사유발생일  거래유형  중개사소재지  등기일자   매도자   매수자 주택유형  
775  25600  4  None      None  중개거래  서울 서초구  None  None  None  다세대  
776  30500  2  None      None  중개거래  서울 서초구  None  None  None  다세대  
777  38600  4     O  22.12.23   직거래    None  None  None  None  다세대  
778  40400  4  None      None  중개거래  서울 서초구  None  None  None  다세대  
779  32800  7  None      None  중개거래  서울 서초구  None  None  None  다세대

---

## 연립다세대 전월세 실거래자료 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명           | 샘플데이터          |
|:---------------|:---------------|:--------------------|
| 지역코드       | 지역코드       | 11110               |
| 법정동         | 법정동         | 청운동              |
| 지번           | 지번           | 1952-05-01 00:00:00 |
| 연립다세대     | 연립다세대명   | 효산빌리지          |
| 건축년도       | 건축년도       | 2010                |
| 층             | 층             | 2                   |
| 전용면적       | 전용면적(㎡)   | 65.59               |
| 년             | 계약년도       | 2022                |
| 월             | 계약월         | 1                   |
| 일             | 일             | 8                   |
| 보증금액       | 보증금(만원)   | 42000               |
| 월세금액       | 월세(만원)     | 0                   |
| 계약구분       | 계약구분       | 갱신                |
| 계약기간       | 계약기간       | 22.01~24.01         |
| 갱신요구권사용 | 갱신요구권사용 | 사용                |
| 종전계약보증금 | 종전계약보증금 | 40000               |
| 종전계약월세   | 종전계약월세   | 0                   |

In [12]:


# 단일 월 조회
# df = api.get_data(
#     property_type="연립다세대",
#     trade_type="전월세",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="연립다세대",
    trade_type="전월세",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  법정동      단지명       지번  건축년도    전용면적  계약년도  계약월  계약일   보증금액  \
7555    11650  반포동       YM    726-3  2015  20.000  2022   12   13   3000   
7556    11650  반포동   반포아트빌라    718-2  2007  34.915  2022   12   24  20000   
7557    11650  반포동     벨뷰반포    712-7  2022  27.300  2022   12    1  30000   
7558    11650  서초동    스위트서초   1601-6  2018  24.430  2022   12   22  26460   
7559    11650  서초동  서초남광쉐르빌  1490-23  2018  30.660  2022   12    5  35700   

      월세금액  층         계약기간 계약구분 갱신요구권사용  종전계약보증금  종전계약월세 주택유형  
7555    65  4  22.12~23.12   신규    None     <NA>    <NA>  다세대  
7556     0  2  23.02~24.12   갱신      사용    20000    <NA>  다세대  
7557     0  3  22.12~24.11   신규    None     <NA>    <NA>  다세대  
7558     0  8  22.12~24.12   갱신    None    25200    <NA>  다세대  
7559     0  6  23.03~25.02   갱신      사용    34000    <NA>  다세대

---

## 단독/다가구 매매 실거래 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명                  | 샘플데이터   |
|:---------------|:----------------------|:-------------|
| 지역코드       | 지역코드              | 11650        |
| 법정동         | 법정동                | 양재동       |
| 지번           | 지번                  | *            |
| 주택유형       | 주택유형(단독/다가구) | 다가구       |
| 건축년도       | 건축년도              | 1998         |
| 대지면적       | 대지면적              | 236.1        |
| 연면적         | 연면적                | 534.18       |
| 년             | 계약년도              | 2022         |
| 월             | 계약월                | 2            |
| 일             | 일                    | 12           |
| 거래금액       | 거래금액(만원)        | 470000       |
| 거래유형       | 중개 및 직거래 여부   | 중개거래     |
| 중개사소재지   | 시군구 단위           | 서울 강남구  |
| 해제사유발생일 | 해제사유발생일        | 22.04.22     |
| 해제여부       | 해제여부              | O            |

In [13]:


# 단일 월 조회
# df = api.get_data(
#     property_type="단독다가구",
#     trade_type="매매",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="단독다가구",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  법정동 주택유형    지번     연면적   대지면적  계약년도  계약월  계약일    거래금액  건축년도  \
126    11650  양재동   단독     *  819.07  335.3  2022   11   10  640000  1988   
127    11650  반포동   단독   7**  597.42  212.1  2022   12   29  590000  1991   
128    11650  방배동   단독   5**  245.88  172.0  2022   12   19  220000  1989   
129    11650  서초동   단독  1***  323.94  614.8  2022   12    5  780000  1990   
130    11650  서초동  다가구  1***  352.61  166.9  2022   12   19  340000  1995   

     해제여부   해제사유발생일  거래유형  중개사소재지   매도자   매수자  
126     O  23.07.31  중개거래  서울 서초구  None  None  
127  None      None  중개거래  서울 강남구  None  None  
128  None      None  중개거래  서울 서초구  None  None  
129     O  23.05.16  중개거래  서울 서초구  None  None  
130  None      None  중개거래  서울 강남구  None  None

## 토지 매매 신고 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명                    | 샘플데이터        |
|:---------------|:------------------------|:------------------|
| 지역코드       | 지역코드                | 11110             |
| 시군구         | 시군구                  | 종로구            |
| 법정동         | 법정동                  | 필운동            |
| 지번           | 지번                    | 1**               |
| 용도지역       | 용도지역                | 제2종일반주거지역 |
| 지목           | 지목                    | 대                |
| 거래면적       | 거래면적(㎡)            | 14                |
| 거래금액       | 거래금액(만원)          | 7000              |
| 구분           | 지분거래구분(지분/공란) | 지분              |
| 년             | 계약년도                | 2015              |
| 월             | 계약월                  | 12                |
| 일             | 일                      | 11                |
| 거래유형       | 중개 및 직거래 여부     | 중개거래          |
| 중개사소재지   | 시군구 단위             | 서울 서초구       |
| 해제사유발생일 | 해제사유발생일          | 21.01.27          |
| 해제여부       | 해제여부                | O                 |

In [15]:


# 단일 월 조회
# df = api.get_data(
#     property_type="토지",
#     trade_type="매매",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="토지",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  시군구  법정동   지번  지목       용도지역    거래면적  거래유형  계약년도  계약월  계약일  \
237    11650  서초구  양재동   8*   대  제2종일반주거지역   285.3  중개거래  2022   12    2   
238    11650  서초구  신원동  2**   전      준주거지역  4090.1   직거래  2022   12    1   
239    11650  서초구  원지동  3**   답     개발제한구역   406.0  중개거래  2022   12    8   
240    11650  서초구  신원동  1**  임야     개발제한구역  7920.0   직거래  2022   12    1   
241    11650  서초구  원지동  3**   전     개발제한구역   816.0  중개거래  2022   12    2   

        거래금액  해제여부   해제사유발생일  중개사소재지 지분거래구분  
237   863000  None      None  서울 강남구   None  
238  1105221  None      None    None   None  
239   100000     O  22.12.22  서울 강남구   None  
240   133452  None      None    None   None  
241    90000  None      None  서울 서초구   None

## 상업업무용 부동산 매매 신고 자료 조회 서비스

### 입력 명세

* [입력 명세](#입력-명세)

### 출력 명세

<div align="center">

| 항목명         | 설명                    | 샘플데이터    |
|:---------------|:------------------------|:--------------|
| 지역코드       | 지역코드                | 11650         |
| 시군구         | 물건 소재 시군구        | 서초구        |
| 법정동         | 물건 소재 법정동        | 반포동        |
| 지번           | 지번                    | 8**           |
| 유형           | 일반/집합 건물 구분     | 집합          |
| 용도지역       | 물건 소재 토지 용도지역 | 제3종일반주거 |
| 건물주용도     | 건물주용도              | 판매          |
| 건축년도       | 건축년도                | 1974          |
| 층             | 거래물건 층(집합건물)   | nan           |
| 대지면적       | 대지권면적              | nan           |
| 건물면적       | 건물 전용/연면적        | 12.46         |
| 구분           | 지분거래 구분           | nan           |
| 년             | 계약년도                | 2022          |
| 월             | 계약월                  | 12            |
| 일             | 계약일                  | 9             |
| 거래금액       | 거래금액                | 67000         |
| 거래유형       | 중개 및 직거래 여부     | 중개거래      |
| 중개사소재지   | 시군구 단위             | 서울 서초구   |
| 해제사유발생일 | 해제사유발생일          | 23.01.10      |
| 해제여부       | 해제여부                | O             |

In [16]:


# 단일 월 조회
# df = api.get_data(
#     property_type="상업업무용",
#     trade_type="매매",
#     sigungu_code="11650",
#     year_month="202212",
# )

# 기간 내 조회
df = api.get_data(
    property_type="상업업무용",
    trade_type="매매",
    sigungu_code="11650",
    start_year_month="202201",
    end_year_month="202212",
)

df.tail()

법정동시군구코드  시군구  법정동    지번 건물유형    건물주용도     용도지역     층  계약년도  계약월  ...  \
758    11650  서초구  서초동  1***   집합       업무     일반상업  <NA>  2022   12  ...   
759    11650  서초구  서초동  1***   집합       업무     일반상업  <NA>  2022   12  ...   
760    11650  서초구  서초동  1***   집합       업무     일반상업  <NA>  2022   12  ...   
761    11650  서초구  서초동  1***   집합  제1종근린생활  제2종일반주거  <NA>  2022   12  ...   
762    11650  서초구  서초동  1***   집합       숙박     일반상업    13  2022   12  ...   

     건축년도   건물면적  대지면적  해제여부  해제사유발생일  거래유형 지분거래구분  중개사소재지   매도자   매수자  
758  2005  10.15   NaN  None     None  중개거래   None  서울 서초구  None  None  
759  2005  10.15   NaN  None     None  중개거래   None  서울 서초구  None  None  
760  2005  10.15   NaN  None     None  중개거래   None  서울 서초구  None  None  
761  2018  40.58   NaN  None     None  중개거래   None  서울 강남구  None  None  
762  2004  31.61   NaN  None     None   직거래   None    None  None  None  

[5 rows x 22 columns]